bower.json

```diff
@@ -8,22 +8,21 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^4.1.0",
-    "purescript-record": "^2.0.0",
-    "purescript-generics-rep": "^6.0.0",
-    "purescript-either": "^4.0.0",
-    "purescript-control": "^4.1.0",
-    "purescript-arrays": "^5.0.0",
-    "purescript-globals": "^4.0.0",
-    "purescript-strings": "^4.0.0",
-    "purescript-lazy": "^4.0.0",
-    "purescript-profunctor": "^4.0.0"
+    "purescript-prelude": "^5.0.0",
+    "purescript-record": "^3.0.0",
+    "purescript-either": "^5.0.0",
+    "purescript-control": "^5.0.0",
+    "purescript-arrays": "^6.0.0",
+    "purescript-strings": "^5.0.0",
+    "purescript-lazy": "^5.0.0",
+    "purescript-profunctor": "^5.0.0",
+    "purescript-js-uri": "https://github.com/purescript-contrib/purescript-js-uri.git#2.0.0"
```

Comment:
`purescript-globals` is removed, so I switch to `purescript-js-uri`. The registry was closed before that was created, so I had to add a github link